### PR TITLE
2 packages from extism/extism at 0.1.0

### PR DESCRIPTION
**Pull request by zshipko**
This pull-request concerns:
- `extism.0.1.0`: Extism bindings
- `extism-manifest.0.1.0`: Extism manifest bindings

---
* Homepage: https://github.com/extism/extism
* Source repo: git+https://github.com/extism/extism.git
* Bug tracker: https://github.com/extism/extism/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/extism/extism.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+maintainer: "Extism Authors <oss@extism.org>"
+authors: "Extism Authors <oss@extism.org>"
+license: "BSD-3-Clause"
+tags: ["topics" "wasm" "plugin"]
+homepage: "https://github.com/extism/extism"
+doc: "https://github.com/extism/extism"
+bug-reports: "https://github.com/extism/extism/issues"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" {>= "3.2" & >= "3.2"}
```
Comment (kit-ty-kate) [inline packages/extism/extism.0.1.0/opam:13]:
```suggestion
  "dune" {>= "3.2"}
```

File: packages/extism/extism.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "ctypes-foreign" {>= "0.18.0"}
```
Comment (kit-ty-kate) [inline packages/extism/extism.0.1.0/opam:14]:
```suggestion
  "ctypes" {>= "0.18.0"}
  "ctypes-foreign" {>= "0.18.0"}
```

File: packages/extism/extism.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  "bigstringaf" {>= "0.9.0"}
+  "ppx_yojson_conv" {>= "0.15.0"}
+  "extism-manifest"
+  "ppx_inline_test" {>= "0.15.0"}
```
Comment (kit-ty-kate) [inline packages/extism/extism.0.1.0/opam:18]:
```suggestion
  "ppx_inline_test" {>= "v0.15.0"}
```

File: packages/extism/extism.0.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+tags: ["topics" "wasm" "plugin"]
+homepage: "https://github.com/extism/extism"
+doc: "https://github.com/extism/extism"
+bug-reports: "https://github.com/extism/extism/issues"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" {>= "3.2" & >= "3.2"}
+  "ctypes-foreign" {>= "0.18.0"}
+  "bigstringaf" {>= "0.9.0"}
+  "ppx_yojson_conv" {>= "0.15.0"}
```
Comment (kit-ty-kate) [inline packages/extism/extism.0.1.0/opam:16]:
```suggestion
  "ppx_yojson_conv" {>= "v0.15.0"}
```

File: packages/extism-manifest/extism-manifest.0.1.0/opam [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+license: "BSD-3-Clause"
+tags: ["topics" "wasm" "plugin"]
+homepage: "https://github.com/extism/extism"
+doc: "https://github.com/extism/extism"
+bug-reports: "https://github.com/extism/extism/issues"
+depends: [
+  "ocaml" {>= "4.14.1"}
+  "dune" {>= "3.2" & >= "3.2"}
+  "ppx_yojson_conv" {>= "0.15.0"}
+  "ppx_inline_test" {>= "0.15.0"}
```
Comment (kit-ty-kate) [inline packages/extism-manifest/extism-manifest.0.1.0/opam:15]:
```suggestion
  "dune" {>= "3.2"}
  "ppx_yojson_conv" {>= "v0.15.0"}
  "ppx_inline_test" {>= "v0.15.0"}
```

**Comment by kit-ty-kate** [comment]
Thanks. Could you return the fixes in the opam file upstream so that they can be taken into account for next time?

**Comment by zshipko** [comment]
Yes I'll do that - thanks for your help!

**Pull request merged**